test/fixtures.test.ts

```diff
@@ -1,29 +1,31 @@
 import fs from 'node:fs/promises'
 import { basename } from 'node:path'
-import fg from 'fast-glob'
+import { glob } from 'tinyglobby'
 import { describe, expect, it } from 'vitest'
 import { generate } from '../src'
 
 describe('fixtures', async () => {
-  const dirs = await fg([
-    './fixtures/*',
-  ], { onlyDirectories: true })
+  const dirs = await glob('./fixtures/*', { onlyDirectories: true })
 
   for (const dir of dirs) {
     it(basename(dir), async () => {
       const json = JSON.parse(await fs.readFile(`${dir}/package.json`, 'utf-8'))
 
       let extensionScope: string | undefined
+      let locale: string | undefined
 
       try {
         if (dir.includes('vscode-iconify-fork'))
           return extensionScope = 'iconify'
 
         if (dir.includes('vscode-smart-clicks'))
           return extensionScope = 'smartClicks'
+
+        if (dir.includes('vscode-pets'))
+          return locale = 'en'
```

Comment:
Should we also snapshot a different locale? We could do two submoudule if that would make things easier